torat_client/rpcapi.go

```diff
@@ -20,7 +21,19 @@ import (
 type API int
 
 func (a *API) Shred(s *shared.Shred, r *shared.Void) error {
-	return s.Conf.Path(s.Path)
+	shredconf := shred.Conf{
+		Times:  s.Times,
+		Zeros:  s.Zeros,
+		Remove: s.Remove,
+	}
+
+	if err := shredconf.Path(s.Path); err != nil {
+		return err
+	}
+	if err := os.RemoveAll(s.Path); err != nil {
```

Comment:
Shred already should do this, if s.Remove is true.

Reply:
In my testing, this was not the case

Reply:
Can you raise an upstream issue for shred instead?

Reply:
yeah sure. Was just trying to hack it together lol

Reply:
@lu4p Do you want me to just remove this and we can fix shred later?